src/app/graphicsUtils.js

```diff
@@ -0,0 +1,31 @@
+define([
+  'esri/graphicsUtils'
+], function (
+  graphicsUtils
+) {
+    return {
+        unionGraphicsIntoExtent: function (graphics) {
```

Comment:
I think it is here since it takes the graphics extent of multiple extents and unions them.